build/wix/mixxx.wxs

```diff
@@ -0,0 +1,429 @@
+    <Property Id="ARPNOREPAIR" Value="yes" Secure="yes" />      <!-- Remove repair -->
+    <Property Id="ARPNOMODIFY" Value="yes" Secure="yes" />      <!-- Remove modify -->
+
+    <!-- Upgrade logic -->
+    <!-- AllowSameVersionUpgrades -> Always upgrade, never allow two versions to be installed next to each other -->
+    <!-- AllowSameVersionUpgrades causes ICE61 which must be ignored -->
+    <MajorUpgrade DowngradeErrorMessage="!(loc.NewerInstalled)" AllowSameVersionUpgrades="yes" />
+
+    <Property Id="VCREDISTINSTALLED">
+      <RegistrySearch Id="VCRedistInstalledSearch"
+                      Root="HKLM"
+                      Key="SOFTWARE\Wow6432Node\Microsoft\VisualStudio\12.0\VC\Runtimes\$(var.Platform)"
+                      Name="Installed"
+                      Type="raw" />
+    </Property>
+    
+    <!-- References the VCRedist package -->
+    <Binary Id="VCRedist" SourceFile="$(var.WINLIBPATH)\vcredist_$(var.Platform).exe" />
+
+    <!-- This will search if mixxx is already installed in another bitWidth -->
+    <Property Id="OTHERBITWIDTHINSTALLED">
+      <RegistrySearch Id="BitWidthInstalledSearch"
+                      Root="HKLM"
+                      Key="SOFTWARE\Wow6432Node\!(loc.ApplicationName)"
+                      Name="installed$(var.notarch)"
+                      Type="raw" />
+    </Property>
+    
+    <!-- This is the main installer sequence run when the product is actually installed -->
+    <InstallExecuteSequence>
+
+      <!-- Determine the install location after the install path has been validated by the installer -->
+      <Custom Action="SetARPINSTALLLOCATION" After="InstallValidate" />
+      <Custom Action="InstallVCRedist" After="InstallValidate">NOT VCREDISTINSTALLED AND NOT Installed AND NOT REMOVE</Custom>
+    </InstallExecuteSequence>
+
+    <!-- Set up ARPINSTALLLOCATION property (http://blogs.technet.com/b/alexshev/archive/2008/02/09/from-msi-to-wix-part-2.aspx) -->
+    <CustomAction Id="SetARPINSTALLLOCATION" Property="ARPINSTALLLOCATION" Value="[INSTALLDIR]" />
+
+    <!-- Install VCRedist -->
+    <CustomAction Id='InstallVCRedist' BinaryKey='VCRedist' ExeCommand='/quiet /install' Return='check' />
+    
+    <!-- License agreement -->
+    <WixVariable Id="WixUILicenseRtf" Value="LICENSE.rtf" />
+    
+    <!-- 
+      Launch conditions
+
+      1. Check minimum OS version (501  = Windows XP)
+         If not, the installation is aborted.
+         By doing the (Installed OR ...) property means that this condition will only be evaluated if the app is being installed and not on uninstall or changing
+
+        Note: Under a Product element, a condition becomes a LaunchCondition entry. 
+        
+        When time will come to remove XP support, simply change 501 to 600 for vista or 601 for seven
+        https://msdn.microsoft.com/en-us/library/aa370556%28v=vs.85%29.aspx
+    -->
+    <Condition Message="!(loc.OS2Old)">
+       <![CDATA[Installed OR (VersionNT >= 501)]]>
```

Comment:
On the one hand, that's fair since we're still supporting Win XP. On the other, I can see people trying to use surplus hardware in poorer countries (where Mixxx is popular) that can't run newer Windows versions. For this reason, my preference would be to put up a warning saying "there is no support for running Mixxx on this OS and it may not work at all" rather than forbid installation. I'll ask in mixxx-devel to see if there's any consensus.